app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -608,16 +608,16 @@ public function newAction(Request $request, AssetsHelper $assetsHelper, Translat
      * @return array|\Symfony\Component\HttpFoundation\JsonResponse|\Symfony\Component\HttpFoundation\RedirectResponse|Response
      */
     public function editAction(
-            Request $request,
-            AssetsHelper $assetsHelper,
-            Translator $translator,
-            RouterInterface $routerHelper,
-            CoreParametersHelper $coreParametersHelper,
-            $objectId,
-            $ignorePost = false,
-            $forceTypeSelection = false
+        Request $request,
+        AssetsHelper $assetsHelper,
+        Translator $translator,
+        RouterInterface $routerHelper,
+        CoreParametersHelper $coreParametersHelper,
+        $objectId,
+        $ignorePost = false,
+        $forceTypeSelection = false
     ) {
-        /** @var \Mautic\EmailBundle\Model\EmailModel $model */
+        /** @var EmailModel $model */
         $model  = $this->getModel('email');
```

Comment:
Just FYI, you can add `EmailModel $model` to the controller method params and Symfony will automatically autowire it so you can get rid of this completely.

Reply:
EmailController extends FormController and FormController extends AbstractStandardFormController and AbstractStandardFormController has 12 param in constructor so if I am adding constructor in EmailController then we have to pass all 12 param while calling parent constructor. also changes will required in each method of this controller. IMO we should leave as it is. what do you think ?

Reply:
@dadarya0 Jan talkingabout add service to method viewAction not construct, something like https://github.com/mautic/mautic/blob/ca03c4b13ef8d2089a922ce93bb6d072e8b35f74/app/bundles/EmailBundle/Controller/EmailController.php#L445

Reply:
ohk , got it , change pushed https://github.com/mautic/mautic/pull/10215/commits/b380561eda895a87ca8f928c68b50a08aecc76e4